Makefile

```diff
@@ -81,7 +80,7 @@ build-and-push-container-windows-%: init-buildx
 	CGO_ENABLED=0 GOOS=windows GOARCH=amd64 go build -a -ldflags='-extldflags="-static" -X="main.version=${STAGINGVERSION}"' -mod vendor -o _output/windows/amd64/local-volume-provisioner.exe ./cmd/local-volume-provisioner
 	$(DOCKER) buildx build --file=./deployment/docker/Dockerfile.Windows --platform=windows/amd64 \
 		-t $(STAGINGIMAGE):$(STAGINGVERSION)_windows_$* \
-		--build-arg BASE_IMAGE=$(call lookup,$*,$(WINDOWS_DISTROS),$(WINDOWS_BASE_IMAGES)) \
+		--build-arg OSVERSION=$* \
```

Comment:
OSVERSION isn't set anywhere where this script runs, please take a look at https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/hack/release.sh#L205

Reply:
thanks, let me change `release.sh` script too, `BASE_IMAGE` depends on `lookup` func, I think that's difficult to maintain.